iroh-net/src/relay/client.rs

```diff
@@ -244,25 +249,92 @@ impl<W: AsyncWrite + Unpin + Send + 'static> ClientWriter<W> {
     }
 }
 
-/// The Builder returns a [`Client`] starts a [`ClientWriter`] run task.
+/// The Builder returns a [`Client`] and a started [`ClientWriter`] run task.
 pub struct ClientBuilder {
     secret_key: SecretKey,
-    reader: RelayReader,
-    writer: FramedWrite<MaybeTlsStreamWriter, DerpCodec>,
-    local_addr: SocketAddr,
+    reader: RelayConnReader,
+    writer: RelayConnWriter,
```

Comment:
Opinions on removing the `Relay` prefixes?

Reply:
Hmmm. I wanted to have something in the name related to `Relay`s.
The old code had a type alias (that effectively had the same job), but it was called `RelayReader`. Are you looking for something shorter/would that be better?

Reply:
Given nothing else is prefixed `Relay*` in this file, I agree I'm going ahead with dropping the prefix :)